doc/DotNet-contrib-examples.rst

```diff
@@ -7,8 +7,8 @@ DeepSpeechWPF
 This examples demonstrates using the .Net Framework DeepSpeech NuGet to build
 a graphical Windows application using DeepSpeech
 
-.. literalinclude:: ../examples/net_framework/DeepSpeechWPF/MainWindow.xaml.cs
+.. literalinclude:: examples/net_framework/DeepSpeechWPF/MainWindow.xaml.cs
    :language: csharp
    :linenos:
 
-Full source code available under `examples/net_framework/DeepSpeechWPF/`.
+Full source code available on `https://github.com/mozilla/DeepSpeech-examples <https://github.com/mozilla/DeepSpeech-examples>`_.
```

Comment:
Maybe to make it easier for people we should deep link into the repo ?

Reply:
The idea is that the examples repo can be adjusted/improved after a release is done. If we deep link here, then tag a release, then we can't change the examples repo anymore because the tag is immutable, and we're back to the same problem again.